variables.tf

```
@@ -4,6 +4,30 @@ variable "datadog_aws_account_id" {
   default     = "464622532012"
 }
 
+variable "enable_kinesis" {
```

looks like this var is not used

---

good call. I think this should be fixed now. I'm using it as a 'count' on the kinesis module itself

---

This var should also be

```suggestion
variable "kinesis_enabled" {
```